### PR TITLE
Add discord badge to readme

### PR DESCRIPTION
**Pull request by Amy-Centrality**
Add a section about the discord community to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,3 +69,18 @@ $ ./target/release/cennznet --dev
...
+
+Join our official CENNZnet Discord server 🤗
+
+* Get CENNZnet technical support 🛠
+* Meet startups and DApp developers 👯‍♂️
+* Learn more about CENNZnet and blockchain 🙌
+* Get updates on CENNZnet bounties and grants 💰
+* Hear about the latest hackathons, meetups and more 👩‍💻
+
+
```
Comment (Amy-Centrality) [inline README.md]: oh there's an extra blank line, I'll remove it

File: README.md [hunk 2]
```diff
@@ -69,3 +69,18 @@ $ ./target/release/cennznet --dev
...
+Join the Discord server by clicking on the badge below!
+
+[![Support Server](https://img.shields.io/discord/591914197219016707.svg?label=Discord&logo=Discord&colorB=7289da&style=for-the-badge)](https://discord.gg/AnB3tRtkJ4)
```
Comment (jordy25519) [inline README.md]: I think the badge needs to be updated to point to our server ID
it displays like this, so guess it needs to be enabled on discord server also
Screen Shot 2021-02-16 at 15 26 34

```suggestion
[![Support Server](https://img.shields.io/discord/801219591636254770.svg?label=Discord&logo=Discord&colorB=7289da&style=for-the-badge)](https://discord.gg/AnB3tRtkJ4)
```

**Review by jordy25519 — COMMENTED** [review]
thanks for the PR 🎉

File: README.md (hunk 2 above)
Comment (Amy-Centrality) [reply README.md]: Oh true yes haha thanks! I'll make the change and enable it

**Comment by Amy-Centrality** [comment]
It's been enabled on the server :D thanks @holygits!

**Review by jordy25519 — APPROVED** [review]
lgtm!

**Review by aliXsed — APPROVED** [review]

**Comment by aliXsed** [comment]
Just a point @Amy-Centrality. Some of your commits seem to be unsigned. Have you completed registering your GPG key?

**Comment by Amy-Centrality** [comment]
Thanks Alex, I'll setup the signing now :D

**Comment by Amy-Centrality** [comment]
Closed this PR because a new one has been made with signed commit

**Pull request closed without merging**